README.md

```diff
@@ -391,12 +391,14 @@ variable to make these changes permanent or use `bat`s
 
 ### Adding new syntaxes / language definitions
 
+To add new syntaxes to your current `bat` installation, the following steps can be performed. 
```

Comment:
I'm okay with putting something like this first, but it kind of repeats what is written below: "To add new syntax definitions, do the following."